tools.go

```diff
@@ -0,0 +1,9 @@
+// +build tools
+
+package tools
```

Comment:
Why do we need this package?

Reply:
It is required to track tool dependencies(https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module)